Sources/PkgConfig/PkgConfig.swift

```diff
@@ -10,12 +10,21 @@
 
 import Utility
 import func POSIX.getenv
+import func POSIX.popen
 
 enum PkgConfigError: ErrorProtocol {
     case CouldNotFindConfigFile
     case ParsingError(String)
 }
 
+/// Get search paths from pkg-config itself.
+/// This is needed because on Linux machines, the search paths can be different
+/// from the standard locations that we are currently searching
+private let pkgConfigSearchPaths: [String] = {
+    let searchPaths = try? POSIX.popen(["pkg-config", "--variable", "pc_path", "pkg-config"])
+    return searchPaths?.characters.split(separator: ":").map(String.init) ?? []
```

Comment:
also can you check if there is any output form popen in case pkg-config is not installed? otherwise we should first confirm there pkg-config is installed before querying it.